### DISCUSSION TITLE
Battery consumption

### DISCUSSION THREAD

**Discussion by garudaonekh** [timmbogner/Farm-Data-Relay-System#3] [General]
hi,
your sensor is using battery? how long can the battery last?
I got a few Lilygo o-it which has builtin battery holder but not yet try it

**Comment by timmbogner** [comment]
The 150mAh battery delivered 2500+ readings on one charge when I tested it at one minute intervals.

**Comment by garudaonekh** [comment]
much better than normal mode.

**Comment by bvwelch** [comment]
I am wondering about gateway battery drain?  In my scenario, I have a long narrow driveway without electricity.  My first thought is to place a lora radio at the far end, and work my way back towards the house/base via several espnow radios.  But in order to save battery  on the lora radio, I was thinking it would ping/poll/transmit every 10 minutes or so and sleep otherwise...

But what is a better way, that is more in step with your system?

Perhaps the lora radio could listen for a ping from the home/base and then sort of sync up and sleep for most of the next 10 minutes and start listening slightly ahead of schedule?

**Reply by timmbogner** [reply]
Scheduling up-time for the entire system is actually an idea I've thought a lot about. It will become a lot easier to add once timekeeping is implemented, which should be soon! My (loose) concept is that the whole system is sleeping until (for example) the top of the hour, then it turns on and does its data stuff for a minute or two before going back to sleep.

**Reply by timmbogner** [reply]
As for running gateways on batteries: they just won't last very long on battery alone. I'm experimenting with solar to see what the minimum size panel/battery would need to be to run a gateway, so I'll let you know what I learn!

**Reply by bvwelch** [reply]
I agree solar will be required - I look forward to your findings. I have a smallish panel, based on one of the Swiss guy's videos, but I haven't tested it yet.

Like you've mentioned previously, long-term I hope to be able to send a photo when a car enters the driveway, etc.

https://stuartsprojects.github.io/2021/11/21/StuartCAM-ESP32CAM-Picture-Transfers-with-LoRa.html

Meanwhile, having to swap battery yearly, or monthly, or even weekly wouldn't be terrible.  A friend of mine uses magnets as quick battery connections.

ps: with timestamps and buffering, as mentioned previously, perhaps gateways could sleep for most of each hour?

**Reply by aviateur17** [reply]
LoRa should be able to transmit for like at least a mile.  How long is your driveway?   Wondering why you are thinking about repeaters between the end of the driveway and the gateway.

**Reply by bvwelch** [reply]
I didn't do a good job of describing my ( half-baked ) plans --  Indeed, I expect the Lora to reach directly from home/base to the end of the driveway (perhaps 1000 ft, with hills, trees, and goats in the way).  So I guess in FDRS lingo, the Lora will be the gateway, and I hope to have, some day, other nodes (espnow?) for camera, radar?  To detect/monitor vehicles approaching home/base.

It may turn out that some number of espnow units, beginning from home-base and directed away from home/base, towards the end of the driveway, is sufficient and no need for Lora, but I already had the Lora boards before I bumped into this cool project.

At this point I'd like to adjust my plans to fit into the FDRS strategy, so thank you for your suggestions.
